upgrade/helpers/constants/environment_config.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from upgrade.helpers.constants import settings
+
+
+FROM_VERSION = settings.UPGRADE.FROM_VERSION
+TO_VERSION = settings.UPGRADE.TO_VERSION
+
+OS = settings.UPGRADE.OS
+
+if OS == "rhel6":
+    CAPSULE_TOOLS_REPO = settings.REPOS.SATTOOLS_REPO.RHEL6
+    RHEV_CAPSULE_AK = settings.UPGRADE.CAPSULE_AK.CAPSULE_AK_RHEL6
+    TOOLS_URL_RHEL = settings.REPOS.SATTOOLS_REPO.RHEL6
+elif OS == "rhel7":
+    CAPSULE_TOOLS_REPO = settings.REPOS.SATTOOLS_REPO.RHEL7
+    RHEV_CAPSULE_AK = settings.UPGRADE.CAPSULE_AK.CAPSULE_AK_RHEL7
+    TOOLS_URL_RHEL = settings.REPOS.SATTOOLS_REPO.RHEL7
+elif OS == "rhel8":
+    CAPSULE_TOOLS_REPO = settings.REPOS.SATTOOLS_REPO.RHEL8
+    RHEV_CAPSULE_AK = settings.UPGRADE.CAPSULE_AK.CAPSULE_AK_RHEL8
+    TOOLS_URL_RHEL = settings.REPOS.SATTOOLS_REPO.RHEL8
+
+
+CLIENT_AK_RHEL6 = settings.UPGRADE.CLIENT_AK.CLIENT_AK_RHEL6
+CLIENT_AK_RHEL7 = settings.UPGRADE.CLIENT_AK.CLIENT_AK_RHEL7
+CLIENT_AK_RHEL8 = settings.UPGRADE.CLIENT_AK.CLIENT_AK_RHEL8
```

Comment:
Use them directly from settings object , need not to save and use from constants.

Also read all of them with lowercase, this looks weird.

Reply:
above 2 comments are for this entire file.